### PR TITLE
Chapter Model

### PR DESCRIPTION
**Pull request by toyhammered**
I feel like I am missing some stuff.

### CODE REVIEW

File: server/db/migrate/20160705041049_create_chapters.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class CreateChapters < ActiveRecord::Migration
+  def change
+    create_table :chapters do |t|
+      # belongs_to manga
+      t.references :manga, index: true
+
+      ## Title Info
+      t.hstore :titles, default: {}, null: false
+      t.string :canonical_title, default: 'en_ja', null: false
```
Comment (NuckChorris) [inline server/db/migrate/20160705041049_create_chapters.rb]: en_jp

File: server/db/migrate/20160705041049_create_chapters.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      t.integer :volume
+      # Total Pages
+      t.integer :length
+      # Synopsis
+      t.text :synopsis
+      # When it was published in magazine
+      t.date :published
+
+
+
```
Comment (NuckChorris) [inline server/db/migrate/20160705041049_create_chapters.rb]: why _three_ empty lines?

File: server/spec/factories/chapters.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+#  synopsis             :text
+#  published            :date
+#  created_at           :datetime         not null
+#  updated_at           :datetime         not null
+#
+
+FactoryGirl.define do
+  factory :chapter do
+    association :manga, factory: :manga
+    titles { {en_jp: Faker::Name.name} }
```
Comment (NuckChorris) [inline server/spec/factories/chapters.rb]: Rubocop gonna bitch at you

File: server/spec/models/chapter_spec.rb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+require 'rails_helper'
+
+RSpec.describe Chapter, type: :model do
+  # subject { create(:chapter) }
+  #
+  # let(:manga) { create(:manga) }
+
+  it { should validate_presence_of(:manga) }
+  it { should validate_presence_of(:number) }
+
```
Comment (NuckChorris) [inline server/spec/models/chapter_spec.rb]: "empty line at end of block" —HoundCI, 2016

Comment (toyhammered) [inline server/spec/models/chapter_spec.rb]: I submitted this 2 days ago prior to Hound bitching me out

**Pull request merged**